Formula/s/solidity.rb

```diff
@@ -23,16 +23,16 @@ class Solidity < Formula
   depends_on "cmake" => :build
   depends_on xcode: ["11.0", :build]
   depends_on "boost"
+  depends_on "z3"
```

Comment:
Can we use Homebrew `fmt` too?

Reply:
yeah, let me update the cmake to use the brewed fmt/nlohmann_json,range-v3